src/components/TableItem.tsx

```diff
@@ -68,8 +68,13 @@ const TableItem: React.FC<TableItemProps> = ({
   return (
     <div className={`${styles.row}`}>
       {/* Displaying properties of the `post` object */}
-      {selectedColumns.uuid && <div className={styles.cell}>{post.uuid}</div>}
-      {selectedColumns.label && <div className={styles.cell}>{post.label}</div>}
+      {selectedColumns.uuid && <div className={styles.cell}>{post.id}</div>}
```

Comment:
```suggestion
      {selectedColumns.uuid && <div className={styles.cell}>{post.uuid}</div>}
```
Sure you didn't mean this?

Reply:
`Property 'uuid' does not exist on type '{ id: string; ...}`

as there is no such field in `tinkerforgeDTO`-type (`post` is `tinkerforgeDTO` and `selectedColumns` is centered on the name of the fields in the `Post` interface)